clouddriver-dcos/src/main/groovy/com/netflix/spinnaker/clouddriver/dcos/provider/agent/DcosInstanceCachingAgent.groovy

```diff
@@ -55,15 +57,17 @@ class DcosInstanceCachingAgent implements CachingAgent, AccountAware {
 
   @Override
   String getAgentType() {
-    "${accountName}/${DcosInstanceCachingAgent.simpleName}"
+    "${accountName}/${clusterName}/${DcosInstanceCachingAgent.simpleName}"
```

Comment:
An alternative to having to create an agent per account per cluster would be to still just create them for a given account, and within the caching agent just query each cluster under the account and cache for all of them. I'm not sure what would be more efficient, though. Any thoughts?